src/tests/test_connection.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from unittest.mock import Mock
+
+from sqlalchemy.engine import Engine
+
+from sql.connection import Connection
+
+
+def test_password_isnt_displayed(monkeypatch):
```

Comment:
What about a test for connection with spaces?

Reply:
there is one already

Reply:
Yeah I saw it later.